server/MainThreadHandler.ts

```diff
@@ -425,7 +427,7 @@ export class MainThreadHandlers {
                     channel: payload.faderIndex,
                     level: parseFloat(payload.level),
                 })
-                mixerGenericConnection.updateOutLevel(payload.faderIndex)
+                mixerGenericConnection.updateOutLevel(payload.faderIndex, 0)

```

Comment:
Settings 2.arg to 0 - will set the Fade time to 0ms is there a reason for doing so?

Reply:
Yes, the intention here is to reduce the time between user input and the desired volume on the sound mixer. Because the user is manually dragging the fader here this is desirable. If the system lags behind what the user is doing, then the user is more likely to overshoot.

Reply:
Ahhh ok :)
That's fine then. I´ll merge it and make a new release.